browser/src/canvas/sections/URLPopUpSection.ts

```diff
@@ -74,29 +74,30 @@ class URLPopUpSection extends HTMLObjectSection {
 				app.map.sendUnoCommand('.uno:JumpToMark?Bookmark:string=' + encodeURIComponent(this.sectionProperties.url.substring(1)));
 		};
 
+		var params: any;
```

Comment:
Nit: if you touch this line anyway, then better to use const or let, 'var' is legacy.